python/whylogs/core/feature_weights.py

```diff
@@ -0,0 +1,76 @@
+import json
+from typing import Any, Dict, Tuple, TypedDict
+
+from whylogs.api.writer import Writers
+from whylogs.api.writer.writer import Optional, Writable, Writer
+
+
+class FeatureWeightResponse(TypedDict):
+    segmentWeights: list
+    metadata: Dict
+
+
+class FeatureWeightDict(TypedDict):
+    segment: Optional[str]
+    weights: Dict[str, float]
+
+
+class FeatureWeight(Writable):
```

Comment:
suggest pluralize this name since it contains a collection of weights? Do we need the other classes above if you have this one?

Reply:
The TypedDict ones actually are mostly there to help with annotation. I could simply remove them and change the annotation.

Reply:
Remove or prefix with underscore if they are internal helpers